src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/Example1.java

```diff
@@ -0,0 +1,27 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NestedForDepth"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.nestedfordepth;
+
+// xdoc section -- start
+public class Example1 {
+  public void myTest() {
+    for(int i=0; i<10; i++) {
+      for(int j=0; j<i; j++) {
+        // violation below, 'Nested for depth is 2 (max allowed is 1).'
+        for(int k=0; k<j; k++) {
```

Comment:
Please look attentively to issue description.
We asking to make Examples same by content and size(line length), to let diff between examples being only in trailing/violation comments.

It will help users to quickly spot a difference in behavior when only config is changed

Reply:
Please make 3 blocks of nested FOR
2 nested 
3 nested 
4 nested 

In first example there will be 2 violations 
In second example there will 1 violation